### DISCUSSION TITLE
stuck in 800x600 resolution

### DISCUSSION THREAD

**Discussion by stoolzo** [fholger/crysis_vrmod#28] [Q&A]
Hi, I cant get the game to run in anything but 800x600 resolution. I seen posts about setting the resolution in Steam but this doesnt work, I set it to run at 1920x1080 on internal and external. I also saw a post about setting it in CFG ini that doesnt work either.

Could it be the problem is that I didnt buy it on steam, its on EA origin and I had to add the game as a none steam game link, but you'd think it would still work. Just stuck in 800x600, cant change ingame either.

Any ideas?

**Comment by L00ncan** [comment]
Me too, also using EA stuck on 800x600 and doesn't detect head movement, menu doesn't move around like it does for others, menu stays static, everything off camera and stuck on 800x600

**Comment by stoolzo** [comment]
this mod has been abandoned sadly, it was eagerly waiting to play this in VR once the bugs had been fixed but sadly thats it, its pretty much unplayable for me :(

**Comment by brunner-itb** [comment]
The mod is not abandoned, but I got the same problem with the gog version now. Looks sad in 800x600, but its still playable. The foliage is the worst tho. Any help anyone?

**Comment by SeantheBomb** [comment]
Is this not a common issue? Surprised how little activity this support thread has. Upon install and first launch of the VR mod it set itself to 800x600

Everything else works fine, but the resolution feels really low (and render distance is low, but I think that's a different issue?)

If I go to settings and try to change the resolution, it just switches itself back to 800x600

I don't even see a resolution that matches the headset

**Comment by fholger** [comment]
The original game's resolution setting has no meaning whatsoever for VR. It always says 800x600 because the game does not understand the actual render resolution.
There is a VR resolution scale slider in the VR settings under miscellaneous.

**Comment by brunner-itb** [comment]
Aaah, you added that in the v1.1 update, amazing. That also fixes the bino crash, thank you so much! Love the mod, great work my man :)
